readme.md

```diff
@@ -388,6 +388,14 @@ The `gh-pages` module writes temporary files to a `node_modules/.cache/gh-pages`
 
 If `gh-pages` fails, you may find that you need to manually clean up the cache directory.  To remove the cache directory, run `node_modules/gh-pages/bin/gh-pages-clean` or remove `node_modules/.cache/gh-pages`.
 
+### Deploying to github pages with custom domain
+
+Modify the deployment line to your deploy script if you use custom domain. This will prevent deployment to remove the domain forom settings in github.
```

Comment:
```suggestion
Modify the deployment line to your deploy script if you use custom domain. This will prevent the deployment from removing the domain settings in GitHub.
```